index.html

```diff
@@ -205,10 +205,10 @@ <h4>Skills</h4>
       </section>
   <section id="contact-section">
     <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
-    <form action="#" id="form">
-      <input type="text" name="fullName" id="fullName"  placeholder="Full Name" required>
+    <form action="https://formspree.io/f/meqwdago" method="post" id="form">
+      <input type="text" name="fullName" id="fullName"  placeholder="Full Name" maxlength="30" required>
       <input type="email" name="email" id="email" placeholder="Email Address" required>
-      <textarea name="message" id="message" cols="30" rows="10"></textarea>
+      <textarea name="message" id="message" cols="30" rows="10" maxlength="500"></textarea>
       <button type="submit">Get in touch</button>
```

Comment:
- [ ] Dear @AsumVictor, good job making the form with a successful submission, But remember that all input fields are required to have the correct associated placeholder text. Please add a placeholder for your text area. 

https://github.com/AsumVictor/Porfolio/assets/69077061/c9aae4d0-07f6-417b-a93d-c2b1b4069f06

- [ ] Besides that, All input fields are required to have the `required` attribute to activate the validation on submit. Please make the `textarea` a required field.

Reply:
Thank you very much